src/database/migrations/create-user.js

```diff
@@ -1,44 +1,58 @@
 export default {
-  up: (queryInterface, Sequelize) => {
-    return queryInterface.createTable('Users', {
-      id: {
-        allowNull: false,
-        autoIncrement: true,
-        primaryKey: true,
-        type: Sequelize.INTEGER
-      },
-      first_name: {
-        type: Sequelize.STRING,
-        allowNull: false,
-      },
-      last_name: {
-        type: Sequelize.STRING,
-        allowNull: false
-      },
-      email: {
-        type: Sequelize.STRING,
-        allowNull: false,
-        unique: true
-      },
-      password: {
-        type: Sequelize.STRING,
-        allowNull: false 
-      },
-      is_admin: {
-        type: Sequelize.BOOLEAN,
-        defaultValue: false
-      },
-      createdAt: {
-        allowNull: false,
-        type: Sequelize.DATE
-      },
-      updatedAt: {
-        allowNull: false,
-        type: Sequelize.DATE
-      }
-    });
+  up(queryInterface, Sequelize) {
+    return queryInterface.sequelize.query('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";')
+      .then(() => queryInterface.createTable('users', {
```

Comment:
Nice one @funmi5, I think @eokwukwe said he prefers `async await` syntax to `.then`

Reply:
@uthdev it doesn't apply here, he meant it for creating functions or methods (controllers)

Reply:
Ok. @funmi5. I feel we could still use that here. But it's fine since it's just one.